juntagrico/admins/subscription_admin.py

```diff
@@ -7,6 +7,7 @@
 
 class SubscriptionAdmin(BaseAdmin):
     form = SubscriptionAdminForm
+    readonly_fields = ('creation_date',)
```

Comment:
wieos read only?

Reply:
Das Feld kann scheinbar nicht bearbeitet werden, weils automatisch gesetzt wird.

Reply:
im nachinein ja schon

Reply:
In der aktuellen Version wird das Feld gar nicht angezeigt, wegen auto_now_add:
https://stackoverflow.com/a/6386240

Reply:
ok dann lasses wir es mal so und wenn extrem viele anfragen kommen um das im nachinein zu ändern dann schauen wir weiter